src/crimson/osd/osd.cc

```diff
@@ -868,7 +823,7 @@ OSD::ShardDispatcher::ms_dispatch(
     return handle_update_log_missing_reply(conn, boost::static_pointer_cast<
       MOSDPGUpdateLogMissingReply>(m));
   default:
-    return seastar::now();
+    return std::nullopt;
```

Comment:
Not related to the change in this PR:
Which cases should the default one cover?  Is it for unsupported msgs?

Reply:
Please refer to the comment of ms_dispatch() interface. It returns std::nullopt for the unsupported message, so that `ChainedDispatchers` will try the next chained dispatcher.